### PR TITLE
Add ability to start calls on Android

### PR DESCRIPTION
**Pull request by grit96**
- Implements `startCall` for Android.
- Adds additional required permission (`CALL_PHONE`).
- Adds `RNCallKeepDidActivateAudioSession` events on Android.

### CODE REVIEW

**Comment by sboily** [comment]
Hello, thank you for your PR. We will check as soon as possible, we are currently busy for 3 days.

**Comment by sboily** [comment]
@danjenkins done.

File: android/build.gradle [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+
 apply plugin: 'com.android.library'
 
 def safeExtGet(prop, fallback) {
     rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 23)
+    compileSdkVersion safeExtGet('compileSdkVersion', 26)
```
Comment (sboily) [inline android/build.gradle]: Why did you changed the version?

Comment (grit96) [reply android/build.gradle]: I had changed it because I was using an API only in 26, but I think I removed it so will change it back to 23.

**Comment by manuquentin** [comment]
Thanks @grit96 !

**Pull request merged**